app/controllers/catalog_controller.rb

```diff
@@ -234,4 +238,8 @@ class CatalogController < ApplicationController
     # mean") suggestion is offered.
     config.spell_max = 5
   end
+  def admin_state_facet
+    return unless current_user && (current_user.curation_concern_creator? || current_user.admin?)
```

Comment:
`can? :create, ScannedResource` works, so I think we should go with that.  We can create a new permission for review later on if creators and reviewers really are separate.